stripper/ze_rollermine_factory_2010_p.cfg

```diff
@@ -0,0 +1,7 @@
+;remove a vending machine at last defense that could be used to get on top of end room. This lead to a delay spot after nuke where whichever team on top of end room could just sit there and wait out round time.
+filter:
+{
+	"classname" "prop_physics_override"
+	"origin" "116 10536 524"
+	"model" "models/props/cs_office/vending_machine.mdl"
+}
```

Comment:
Can we keep this prop and just expand the nuke trigger to above the room instead? The vending machine can play a big part in the end sequence when people try to get others stuck with it, which fits the maps theme.